CRM/Core/BAO/FinancialTrxn.php

```diff
@@ -669,7 +669,9 @@ public static function createDeferredTrxn($lineItems, $contributionDetails, $upd
         }
       }
       $accountRel = key(CRM_Core_PseudoConstant::accountOptionValues('account_relationship', NULL, " AND v.name LIKE 'Income Account is' "));
-      // TODO: Call hook to alter $deferredRevenues
+
+      CRM_Utils_Hook::alterDeferredRevenueItems($deferredRevenues);
```

Comment:
let's add more args here - $contributionDetails, $update, $context - in order to help anyone implementing hook so they know what this function does